code/_core/mob/living/advanced/human/npc/syndicate.dm

```diff
@@ -76,9 +76,8 @@
 
 	add_outfit(outfit_to_use)
 
-	var/picked_weapon = pick(possible_weapons)
-
 	/*
+	var/picked_weapon = pick(possible_weapons)
```

Comment:
Unrelated, but this bit was throwing a warning because it was unused. Annoying.

Reply:
oh yeah